docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+
+services:
+  ia-homeoffice-integration-api:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    image: hmcts/ia-homeoffice-integration-api
+    container_name: ia-homeoffice-integration-api
+    environment:
+      # these environment variables are used by java-logging library
+          JAVA_OPTS:
+          # these environment variables are used by java-logging library
+          JSON_CONSOLE_PRETTY_PRINT:
+          ROOT_APPENDER:
+          ROOT_LOGGING_LEVEL:
+          REFORM_SERVICE_TYPE:
+          REFORM_SERVICE_NAME:
+          REFORM_TEAM:
+          REFORM_ENVIRONMENT:
+          LOGBACK_DATE_FORMAT:
+          LOGBACK_REQUIRE_THREAD:
+          LOGBACK_REQUIRE_ALERT_LEVEL: "false"
+          LOGBACK_REQUIRE_ERROR_CODE: "false"
+          # Application environment variables
+          CCD_GW_URL: http://ccd-api-gateway-web:3453
+          IDAM_URL: http://idam-api:8080
+          IA_IDAM_REDIRECT_URI: http://idam-api
+          S2S_URL: http://service-auth-provider-api:8080
+          CCD_URL: http://ccd-data-store-api:4452
+          DM_URL: http://dm-store:8080
+          IA_IDAM_CLIENT_ID:
+          IA_IDAM_SECRET:
+          IA_S2S_MICROSERVICE:
+          IA_S2S_SECRET:
+          IA_SYSTEM_USERNAME:
+          IA_SYSTEM_PASSWORD:
+          IA_EM_STITCHING_ENABLED:
+          IA_DOCMOSIS_ENABLED:
+          IA_SUBMIT_HEARING_REQUIREMENTS_ENABLED:
```

Comment:
please remove things we don't know we need.

Reply:
Removed other variables